Assets/HoloToolkit/Input/Scripts/InputSources/GesturesInput.cs

```diff
@@ -89,80 +93,120 @@ protected override void OnDestroy()
 
                 gestureRecognizer.Dispose();
             }
+            if (navigationGestureRecognizer != null)
+            {
+                navigationGestureRecognizer.NavigationStartedEvent -= OnNavigationStartedEvent;
+                navigationGestureRecognizer.NavigationUpdatedEvent -= OnNavigationUpdatedEvent;
+                navigationGestureRecognizer.NavigationCompletedEvent -= OnNavigationCompletedEvent;
+                navigationGestureRecognizer.NavigationCanceledEvent -= OnNavigationCanceledEvent;
+
+                navigationGestureRecognizer.Dispose();
+            }
+        }
 
-            base.OnDestroy();
+        protected virtual void OnDisable()
+        {
+            StopGestureRecognizer();
+        }
+
+        protected virtual void OnEnable()
+        {
+            if (RecognizerStart == RecognizerStartBehavior.AutoStart)
+            {
+                StartGestureRecognizer();
+            }
+        }
+
+        /// <summary>
+        /// Make sure the gesture recognizer is off, then start it.
+        /// Otherwise, leave it alone because it's already in the desired state.
+        /// </summary>
+        public void StartGestureRecognizer()
+        {
+            if (gestureRecognizer != null && !gestureRecognizer.IsCapturingGestures())
+            {
+                gestureRecognizer.StartCapturingGestures();
+            }
+            if (navigationGestureRecognizer != null && !navigationGestureRecognizer.IsCapturingGestures())
+            {
+                navigationGestureRecognizer.StartCapturingGestures();
+            }
+        }
+
+        /// <summary>
+        /// Make sure the gesture recognizer is on, then stop it.
+        /// Otherwise, leave it alone because it's already in the desired state.
+        /// </summary>
+        public void StopGestureRecognizer()
+        {
+            if (gestureRecognizer != null && gestureRecognizer.IsCapturingGestures())
+            {
+                gestureRecognizer.StopCapturingGestures();
+            }
+            if (navigationGestureRecognizer != null && navigationGestureRecognizer.IsCapturingGestures())
+            {
+                navigationGestureRecognizer.StopCapturingGestures();
+            }
         }
 
-        private void OnTappedEvent(InteractionSourceKind source, int tapCount, Ray headRay)
+        protected void OnTappedEvent(InteractionSourceKind source, int tapCount, Ray headRay)
         {
-            SourceClickEventArgs args = new SourceClickEventArgs(this, 0, tapCount);
-            RaiseSourceClickedEvent(args);
+            inputManager.RaiseSourceClicked(this, 0, tapCount);
```

Comment:
Well that works for a scenario when looking at the same hologram when interacting. But there are scenarios where I could look away but still be interacting with the previously gazed at hologram. Especially in manipulation like scenarios.